index.html

```diff
@@ -249,6 +249,30 @@ <h3>Sampling and Reporting Rate</h3>
     defined by {{PressureSource}}. A [=pressure source=] can make use of data
     fusion with data from additional sources if that provides better results.
   </p>
+  <p>
+    The telemetry information provided by a [=pressure source=] is represented
+    in this specification as a <dfn>pressure source sample</dfn>, a [=struct=]
+    consisting of the following [=struct/items=]:
+    <ul>
+      <li>
+        <dfn data-dfn-for="pressure source sample">data</dfn>: [=contributing
+        factors=] obtained from the underlying hardware or operating system.
+      </li>
+      <li>
+        <dfn data-dfn-for="pressure source sample">timestamp</dfn>: the
+        [=unsafe shared current time=] when [=pressure source sample/data=] was
+        obtained.
+        <aside class="note">
+          <p>
+            The purpose of [=pressure source sample/timestamp=] is to use the
+            same [=monotonic clock/unsafe current time=] for a sample across
+            all globals and invocations of the [=data collection=] algorithm
+            which are processing the same [=pressure source sample=].
+          </p>
+        </aside>
+      </li>
+    </ul>
+  </p>
```

Comment:
:+1:

Reply:
Maybe, we could "in between the lines" tell that this struct, will be, after going through the algorithms, translated to  "pending sample descriptor" ?

Just to avoid terms confusion or connecting dots.

Reply:
I've just pushed a new commit that removes "pending sample descriptor" altogether and uses "pressure source sample" for both real and virtual pressure sources (since a virtual pressure source is a pressure source after all).